android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/everscale/TestEverscaleSigner.kt

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2017-2022 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+package com.trustwallet.core.app.blockchains.everscale
+
+import com.google.protobuf.ByteString
+import com.trustwallet.core.app.utils.Numeric
+import com.trustwallet.core.app.utils.toHexByteArray
+import com.trustwallet.core.app.utils.toHexBytes
+import com.trustwallet.core.app.utils.toHexBytesInByteString
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import wallet.core.java.AnySigner
+import wallet.core.jni.CoinType.EVERSCALE
+import wallet.core.jni.proto.Everscale
+import wallet.core.jni.proto.Everscale.SigningOutput
+
+class TestEverscaleSigner {
+
+    init {
+        System.loadLibrary("TrustWalletCore")
+    }
+
+    @Test
+    fun testSign() {
+        val transferMessage = Everscale.Transfer.newBuilder().apply {
```

Comment:
It was a deployment-only message. Replaced it with a regular transfer, and refactored proto so that it was impossible to create such messages, they are not needed.